src/types/annotator.js

```diff
@@ -59,8 +59,8 @@
  *
  * @typedef AnchorPosition
  * @prop {string} tag - annotation tag
- * @prop {number} top
- * @prop {number} bottom
+ * @prop {number} top - in pixel
```

Comment:
This answers the question about _units_ but what about coordinate system? Are these relative to the top of the document? The top of the viewport? Something else?

Reply:
Good point. Zero is the top of the viewport. Elements that are above the viewport have negative values.
Would that be enough?

Reply:
Yes. Saying "viewport relative" or similar would be fine.